pkg/controller/storagecluster/storagecluster_controller_test.go

```diff
@@ -711,3 +712,48 @@ func TestNodeTopologyMapLabelSelector(t *testing.T) {
 	}
 	assert.Equal(t, nodeTopologyMap, sc.Status.NodeTopologies)
 }
+
+//TestStorageClusterOnMultus tests if multus configurations in StorageCluster are successfully applied to CephClusterCR
+func TestStorageClusterOnMultus(t *testing.T) {
+	// When only public network is specified.
+	createStorageClusterWithMultus(t,"public-network","")
```

Comment:
```suggestion
	createStorageClusterWithMultus(t, "public-network", "")
```

Reply:
Is it me or the linter should catch this.

Reply:
Looks like linter isn't catching a lot of these issues.